scene/2d/navigation_region_2d.cpp

```diff
@@ -420,12 +434,42 @@ void NavigationRegion2D::_region_update_transform() {
 
 #ifdef DEBUG_ENABLED
 void NavigationRegion2D::_update_debug_mesh() {
-	Vector<Vector2> navigation_polygon_vertices = navigation_polygon->get_vertices();
-	if (navigation_polygon_vertices.size() < 3) {
+	if (!is_inside_tree()) {
+		_free_debug();
 		return;
 	}
 
-	const NavigationServer2D *ns2d = NavigationServer2D::get_singleton();
+	NavigationServer2D *ns2d = NavigationServer2D::get_singleton();
+	RenderingServer *rs = RenderingServer::get_singleton();
```

Comment:
Still makes sense for those to be const IMO.
```suggestion
	const NavigationServer2D *ns2d = NavigationServer2D::get_singleton();
	const RenderingServer *rs = RenderingServer::get_singleton();
```